src/widgets/Xpert.jsx

```diff
@@ -29,27 +40,36 @@ const Xpert = ({
     dispatch(updateSidebarIsOpen(isOpen));
   };
 
-  useEffect(() => {
-    dispatch(getLearningAssistantChatSummary(courseId));
-  }, [dispatch, courseId]);
+  const [decision] = useAuditTrialExperimentDecision();
+  const { enabled: experimentEnabled, variationKey } = decision || {};
+
+  // if a user is not part of the experiment, not part of the control, or not eligible for upgrade, they should
+  // still see xpert
+  const shouldDisplayXpert = (
+    !experimentEnabled
+    || !(variationKey === OPTIMIZELY_AUDIT_TRIAL_LENGTH_EXPERIMENT_VARIATION_KEYS.CONTROL)
+    || isStaff
+  );
 
   return isEnabled ? (
     <CourseInfoProvider value={courseInfo}>
       <ExperimentsProvider>
-        <>
-          <ToggleXpert
-            courseId={courseId}
-            isOpen={sidebarIsOpen}
-            setIsOpen={setSidebarIsOpen}
-            contentToolsEnabled={contentToolsEnabled}
-          />
-          <Sidebar
-            courseId={courseId}
-            isOpen={sidebarIsOpen}
-            setIsOpen={setSidebarIsOpen}
-            unitId={unitId}
-          />
-        </>
+        { shouldDisplayXpert() ? (
```

Comment:
Since the above change was made, this would need to be:
```suggestion
        { shouldDisplayXpert ? (
```
instead.